engines/stark/console.cpp

```diff
@@ -588,6 +587,8 @@ bool Console::Cmd_ChangeLocation(int argc, const char **argv) {
 }
 
 bool Console::Cmd_ChangeChapter(int argc, const char **argv) {
+	static const int32 maxChapterValue = 150;
```

Comment:
Other chapter checks use 999 as the max value. I see no reason to be very restrictive here, console commands are for developers to debug the game.